lib/utils/nodeFieldIndices.mjs

```diff
@@ -0,0 +1,118 @@
+import getAtRuleParams from './getAtRuleParams.mjs';
+import getRuleSelector from './getRuleSelector.mjs';
+
+import { isObject, isString } from './validateTypes.mjs';
+
+/** @import {AtRule, Declaration, Rule} from 'postcss' */
+
+/**
+ * @param {AtRule} atRule
+ * @returns {number}
+ */
+export function atRuleParamIndex(atRule) {
+	let index = atRuleRawsAfterNameIndex(atRule);
+
+	if (atRule.raws.afterName) {
+		index += atRule.raws.afterName.length;
+	}
+
+	return index;
```

Comment:
[suggestion] This is just my preference, but how about shortening the function body code like this?

```suggestion
	const index = atRuleRawsAfterNameIndex(atRule);

	return index + (atRule.raws.afterName?.length ?? 0);
```